work around regression in 2.40.2 where BWRAP_EXECUTABLE is referenced
even when ENABLE_BUBBLEWRAP_SANDBOX is OFF


--- webkitgtk-2.40.2/Source/WTF/wtf/glib/Sandbox.cpp-	2023-05-29 08:59:44.764171000 +0000
+++ webkitgtk-2.40.2/Source/WTF/wtf/glib/Sandbox.cpp	2023-06-01 19:19:54.804669775 +0000
@@ -28,6 +28,10 @@
 
 #include <glib.h>
 
+#ifndef BWRAP_EXECUTABLE
+#define BWRAP_EXECUTABLE "/usr/bin/false"
+#endif
+
 namespace WTF {
 
 bool isInsideFlatpak()
